patroni/daemon.py

```diff
@@ -1,16 +1,39 @@
+"""Daemon processes abstraction module.
+
+This module implements abstraction classes and functions for creating and managing daemon processes in Patroni through
+Web API and CLI interfaces.
+"""
 from __future__ import print_function
 
 import abc
 import os
 import signal
 import sys
+from typing import Any
+from patroni.config import Config
+from patroni.validator import Schema
```

Comment:
please move it to right after `from threading import Lock`.

Also it would be nice to separate imports of Config and Schema with the newline.